clients/camunda-spring-boot-starter/src/main/java/io/camunda/client/spring/configuration/CredentialsProviderConfiguration.java

```diff
@@ -101,17 +101,11 @@ private CredentialsProvider buildOAuthCredentialsProvider(
                 camundaClientProperties.getAuth().getClientAssertion().getKeystoreKeyPassword());
 
     maybeConfigureIdentityProviderSSLConfig(credBuilder, camundaClientProperties);
-    try {
-      return credBuilder.build();
-    } catch (final Exception e) {
-      LOG.warn(
-          "Failed to configure oidc credential provider, falling back to use no authentication, cause: {}",
-          e.getMessage());
-      if (LOG.isDebugEnabled()) {
-        LOG.debug(e.getMessage(), e);
-      }
-      return new NoopCredentialsProvider();
-    }
+    return credBuilder.build();
```

Comment:
Defaulting to a noop credentials provider is not useful anymore, this should also be backported to 8.8